packages/react-hooks/src/useControllableValue.ts

```diff
@@ -0,0 +1,51 @@
+import * as React from 'react';
+
+export type ChangeCallback<TElement extends HTMLElement, TValue> = (
+  ev: React.FormEvent<TElement>,
```

Comment:
If I remember right there may be some cases where the event is actually undefined (like if a change callback is used for manual updates). So the typings should probably allow for that.